docs/api.md

```diff
@@ -1441,6 +1463,29 @@ await page.evaluate(() => matchMedia('print').matches);
 // → false
 ```
 
+#### page.emulateNetworkConditions(networkConditions) 
```

Comment:
@jschfflr Thanks for working on this feature. As someone looking to switch to using this API to emulate offline testing, I wasn't sure how to transition from `Network.emulateNetworkConditions`, which accepted an `offline` property to this API. Is the calling `page.setOfflineMode(true)` all that's needed?

Unfortunately `setOfflineMode` docs also don't go into detail.

It might be worth mentioning how this should work. If you can clarify I'm happy to update the docs.

Reply:
Yes, `page.setOfflineMode(true)` is all you need. I'll update the documentation accordingly.

Reply:
Let me know if that helps or if I should further clarify their relationship: https://github.com/puppeteer/puppeteer/pull/7446

Reply:
I'll close this comment to move the discussion to the pull request.